src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -797,20 +793,17 @@ def helm_install_release(resource_manager, chart_path, subscription_id, kubernet
             his_endpoint = his_endpoint + f"discovery?location={location}&api-version=1.0-preview"
             relay_endpoint = arm_metadata["suffixes"]["relayEndpointSuffix"]
             active_directory = arm_metadata["authentication"]["loginEndpoint"]
+            kube_aad_endpoint = f"{aad_identity_principal_id}.k8sproxysvc.connectrp.azs"
             cmd_helm_install.extend(
                 [
-                    "--set", "systemDefaultValues.azureResourceManagerEndpoint={}".
-                    format(resource_manager),
-                    "--set", "systemDefaultValues.azureArcAgents.config_dp_endpoint_override={}".
-                    format(config_endpoint),
-                    "--set", "systemDefaultValues.clusterconnect-agent.notification_dp_endpoint_override={}".
-                    format(notification_endpoint),
-                    "--set", "systemDefaultValues.clusterconnect-agent.relay_endpoint_suffix_override={}".
-                    format(relay_endpoint),
-                    "--set", "systemDefaultValues.clusteridentityoperator.his_endpoint_override={}".
-                    format(his_endpoint),
-                    "--set", "systemDefaultValues.activeDirectoryEndpoint={}".
-                    format(active_directory)
+                    "--set", "global.kubeAadEndpoint={}".format(kube_aad_endpoint),
```

Comment:
In this set of changes too, wrap the lines to adhere to the requirements.  You can copy the existing lines - I had gone through the entire code and fixed these.  Your change is reintroducing them :)

Reply:
Hi Narayan, I don't get the warnings about 120 line width, so wondering what you configured to see those errors in azdev style.